src/pal/inc/pal.h

```diff
@@ -4679,7 +4679,7 @@ enum {
 //
 typedef struct _RUNTIME_FUNCTION {
     DWORD BeginAddress;
-#ifdef _AMD64_
+#if RUNTIME_FUNCTION_HAS_ENDADDRESS
```

Comment:
@janvorli As I understand, the following code in ``CEEJitInfo::allocUnwindInfo`` should be paired with the above ``_RUNTIME_FUNCTION`` definition:
```
#if RUNTIME_FUNCTION_HAS_ENDADDRESS
    pRuntimeFunction->EndAddress        = currentCodeOffset + endOffset;
#endif
```

For me, it was hard to find this relationship when the above definition is written using ``_TARGET_AMD64_``.

Therefore, I would like not to revert this change if there is no further issue. Could you let me know your opinion?